src/librustc_back/target/apple_ios_base.rs

```diff
@@ -99,6 +99,7 @@ pub fn opts(arch: Arch) -> Result<TargetOptions, String> {
         executables: true,
         pre_link_args,
         has_elf_tls: false,
+        exe_allocation_crate: None,
```

Comment:
@kali can you add a comment noting that this new line can be removed after Rust upgrades to a sufficiently new version of jemalloc (and mention #45262 in the comment)?

Thanks!